README.md

```diff
@@ -108,11 +108,15 @@ Build package
 ./gradlew clean test shadowJar
 ```
 
-Setup environment variables
+Setup local bucket
 
 ```bash
-export ECS_ACCOUNT_ID=dev
-export AWS_PROFILE=befaster    # pre-configured profile contained in ~/.aws/credentials
+export AWS_PROFILE=befaster                      # pre-configured profile contained in ~/.aws/credentials
+
+minio config host add myminio http://192.168.1.190:9000 local_test_access_key local_test_secret_key
```

Comment:
This dependency should be covered by the `local-s3` dependency. We should use that instead.